libpromises/mod_common.c

```diff
@@ -54,8 +54,8 @@ static const char *const POLICY_ERROR_VARS_CONSTRAINT_DUPLICATE_TYPE =
     "redefine with %s";
 static const char *const POLICY_ERROR_VARS_PROMISER_NUMERICAL =
     "Variable promises cannot have a purely numerical promiser (name)";
-static const char *const POLICY_ERROR_VARS_PROMISER_RESERVED =
-    "Variable promise is using a reserved name";
+static const char *const POLICY_ERROR_VARS_PROMISER_INVALID =
+    "Variable promise is using an invalid name (promiser)";
```

Comment:
All the other errors say *promiser (name)*, I think we should be consistent here.

Reply:
I'd like to change them all to be name (promiser) :) Focus on the easy terminology for new users.

Reply:
Changed the other error messages to also be `name (promiser)`.